administration/monitoring.md

```diff
@@ -1,29 +1,35 @@
 ---
+title: Monitor data pipelines
 description: Learn how to monitor your Fluent Bit data pipelines
 ---
 
 <img referrerpolicy="no-referrer-when-downgrade" src="https://static.scarf.sh/a.png?x-pxid=e9ca51eb-7faf-491d-a62e-618a21c94506" />
 
-# Monitoring
+# Monitor data pipelines
 
-Fluent Bit comes with built-it features to allow you to monitor the internals of your pipeline, connect to Prometheus and Grafana, Health checks  and also connectors to use external services for such purposes:
+Fluent Bit includes features for monitoring the internals of your pipeline, and for
+connecting to Prometheus and Grafana, Health checks, and connectors to use external
+services:
```

Comment:
```suggestion
Fluent Bit includes features for monitoring the internals of your pipeline, 
connecting to Prometheus and Grafana, Health checks, and connectors to use external
services:
```

multiple ands in there feels awkward

Reply:
Mmm, yeah -- good callout! I'll change to read:

>Fluent Bit includes features for monitoring the internals of your pipeline, in
>addition to connecting to Prometheus and Grafana, Health checks, and connectors to
>use external services: